generic/src/main/java/net/swofty/types/generic/gui/inventory/SkyBlockShopGUI.java

```diff
@@ -221,16 +226,23 @@ public void run(InventoryPreClickEvent e, SkyBlockPlayer player) {
                         return;
                     }
 
-                    double purse = player.getDataHandler().get(DataHandler.Data.COINS, DatapointDouble.class).getValue();
-                    if (price > purse) {
-                        player.sendMessage("§cYou don't have enough coins!");
-                        return;
+                    for (ShopPrice price1 : price) {
```

Comment:
Variables should not be named “price1”, if price contains multiple prices then why not just call it “prices” and then rename price1 to price?